src/Cache/Manager.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Jahuty\Cache;
+
+use Jahuty\Action\{Action, Show};
+use Jahuty\Client;
+use Jahuty\Resource\Resource;
+use Psr\SimpleCache\CacheInterface as Cache;
+
+class Manager
+{
+    private $cache;
+
+    private $client;
+
+    private $ttl;
+
+    public function __construct(Client $client, Cache $cache, $ttl = null)
+    {
+        if ($ttl !== null &&
```

Comment:
consider moving the duplicated logic to a private method (validateTtlsParams?)

Reply:
Ha, shoot! I didn't see your comment until just now, after my latest `push`, and I totally agree! I had to make some changes to the time-to-live validation logic, and after about the third time, I realized I was being dumb.

I added a `Jahuty\Ttl\Ttl` class, which performs validation and provides some convenience methods like `isNull()` and `toSeconds()`. 

Users can still pass `null`, an `int`, or a `DateInterval` value as options to `Jahuty\Client` or `Jahuty\Service\Snippet`. Once inside the library, though, all those optional `$ttl` arguments are now required `Ttl` instances.